### PR TITLE
Remove API key from log statement

### PR DESCRIPTION
**Pull request by aaronfay**
The API key is being obscured when the polling requests are made, but still plainly visible when the suite is executed, this removes the API key from the logs.

### CODE REVIEW

File: run-suite-task/ghost-inspector/services/index.ts [hunk 1]
```diff
@@ -46,11 +46,11 @@ export class Suite {
 
   async execute (): Promise<boolean> {
     // TODO: test network failure
-    console.log('POSTing execute request with body', this.suiteExecuteUrl, this.body)
+    const safeUrl = `${this.suiteResultsBaseUrl}/${suiteResultId}/?apiKey=***`
```
Comment (justinklemm) [inline run-suite-task/ghost-inspector/services/index.ts]: This doesn't seem like the right URL... I think you copy/pasted from the `poll()` function, but the endpoints are different. Shouldn't it be `${this.suiteExecuteBaseUrl}/${this.request.suiteId}/execute/?apiKey=`?

Also might be cleaner to reuse the `suiteExecuteUrl` function and just add a `obfuscate` boolean parameter that hides the API key when true.

Comment (aaronfay) [reply run-suite-task/ghost-inspector/services/index.ts]: Oh good call, I was being lazy there.

**Comment by aaronfay** [comment]
Visual Studio Marketplace would not accept a status badge from CircleCI for the extension public page, they have a specific list of public status badge URLs that they will accept. We could certainly move the build to CircleCI, but we would lose the ability to display the status badge on the [extension page](https://marketplace.visualstudio.com/items?itemName=ghost-inspector.ghost-inspector-vsts-extension).

**Comment by justinklemm** [comment]
Ah, right. Forgot about that. The TravisCI setup is fine, but we should make sure it's duplicated in CircleCI so we get alerted. In this case, TravisCI failed but Circle didn't?

**Comment by aaronfay** [comment]
Yeah, the tests are passing in Circle, still digging into Travis.

**Comment by aaronfay** [comment]
Heh, Travis doesn't like my shortcut either ;) fixing...

File: run-suite-task/ghost-inspector/services/index.ts (hunk 1 above)
Comment (aaronfay) [reply run-suite-task/ghost-inspector/services/index.ts]: This is fixed, [tests added](https://github.com/ghost-inspector/ghost-inspector-vsts-extension/pull/9/files#diff-82b3f263f3caec26e54bcd2f1a8f66d3R26)

**Comment by aaronfay** [comment]
Fixed

**Pull request merged**